src/Archipelago/ArchipelagoIntegration.cs

```diff
@@ -127,6 +127,17 @@ public class ArchipelagoIntegration {
 
                 SetupDataStorage();
 
+                // start inventory items have a location ID of -2, add them to a dict so we can use them for first steps
+                foreach (NetworkItem item in session.Items.AllItemsReceived) {
+                    if (item.Location == -2) {
+                        string itemName = session.Items.GetItemName(item.Item);
+                        if (ItemLookup.Items.ContainsKey(itemName)) {
+                            itemName = ItemLookup.Items[itemName].ItemNameForInventory;
+                        }
+                        ItemRandomizer.AddStringToDict(Hints.StartInventoryItems, itemName);
```

Comment:
```suggestion
                                                   ItemRandomizer.AddStringToDict(Hints.StartInventoryItems, 
ItemLookup.Items[itemName].ItemNameForInventory);
                        }
```
not that this will ever happen or break if it does, but this'll prevent any unknown item names from being added